numpy/distutils/misc_util.py

```diff
@@ -92,7 +93,8 @@ def get_num_build_jobs():
 
     """
     from numpy.distutils.core import get_distribution
-    envjobs = int(os.environ.get("NPY_NUM_BUILD_JOBS", 1))
+    cpu_count = multiprocessing.cpu_count()
```

Comment:
don't use cpu_count it doesn't respect affinity, use `len(os.sched_getaffinity(0))` when available.

Reply:
Will do, but this should really be implemented in Python.

Reply:
that is my opinion too, but the best I could get from them is a docstring update ...